### PR TITLE
patch to proper overwrite 'label_translator_strategy' from yml config…

### LINKED ISSUE #3190: can not activate sonata.admin.label.strategy.underscore
when i use:

``` yml
sonata_admin:
    security:
        handler: sonata.admin.security.handler.role
    admin_services:
        label_translator_strategy: sonata.admin.label.strategy.underscore
```

i got the following error:
bildschirmfoto 2015-08-21 um 14 55 12

### PR DESCRIPTION
**Pull request by nnmer**
… with 'sonata_admin.admin_services.label_translator_strategy'

Closes #3190

## Changelog
```markdown

### Fixed
- patch to proper overwrite 'label_translator_strategy' from yml config with **sonata_admin.admin_services.label_translator_strategy**
```

## Subject
when you try to setup **label_translator_strategy** globaly fro all admins, other that copy past for each admin in it's configuration, the option is to use  '**sonata.admin_services.sonata_admin.admin_services**' in global configuration for sonata_Admin. but it doesn't work, first is same errors as in issue #3190, but then it was not really applying the new strategy and still was using 'native'

### CODE REVIEW

**Comment by nnmer** [comment]
can somebody review?
the tests failed not because of current commit changes

**Comment by OskarStark** [comment]
I restarted the build, if its still failing, please rebase your PR

**Comment by nnmer** [comment]
@OskarStark  tests still fails at 'Unrecognized options "sonata_post_admin, sonata_news_admin" under "sonata_admin.admin_services"'

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -12,21 +12,19 @@
 
 #### Sonata packages
 
-```bash
```
Comment (greg0ire) [inline .github/ISSUE_TEMPLATE.md]: Looks like you messed up somewhere

Comment (nnmer) [reply .github/ISSUE_TEMPLATE.md]: it is look like last commit with that changes was at Aug 8 by SonataCI

Comment (core23) [reply .github/ISSUE_TEMPLATE.md]: You picked the wrong branch for this PR. You must pick `3.x` instead of the `master` branch

Comment (nnmer) [reply .github/ISSUE_TEMPLATE.md]: @core23  at the top of the page you can see
nnmer  wants to merge 63 commits into sonata-project:**3.x** from nnmer:patch-config-yml-admin_services-param

so i don't think i miss the brach

Comment (nnmer) [reply .github/ISSUE_TEMPLATE.md]: anyway, i will try to resubmit the PR again

**Comment by nnmer** [comment]
Close, resubmitted at #4637

**Pull request closed without merging**